test/NuGet.Core.Tests/NuGet.Packaging.Test/PackageArchiveReaderTests.cs

```diff
@@ -5,7 +5,9 @@
 using System.IO;
 using System.IO.Compression;
 using System.Linq;
+#if IS_SIGNING_SUPPORTED
```

Comment:
These `#if`'s resolve IDE warnings that some `using` statements are unnecessary under some TFM's.